.github/workflows/build_push_and_restart_cas.yml

```diff
@@ -3,8 +3,8 @@ name: build_push_and_restart_cas
 on:
   push:
     branches:
-      ##MASKED## - main
-      ##MASKED## - "release-candidate"
+      - main
+      - "release-candidate"
```

Comment:
Are the double quotes needed? Not a huge deal, just looks a little odd since they're not used for the other branch names.

Reply:
Also, at [L31](https://github.com/ceramicnetwork/ceramic-anchor-service/pull/477/files#diff-2a4bf41db23fc4c3523c50fbe7a38582e5eef6e930a028845cbf3b4aa52d7944R31) and [L35](https://github.com/ceramicnetwork/ceramic-anchor-service/pull/477/files#diff-2a4bf41db23fc4c3523c50fbe7a38582e5eef6e930a028845cbf3b4aa52d7944R35), are both `ref` and `base_ref` needed?

Reply:
For [reference](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context) regarding the `github` context.